test-framework/sudo-compliance-tests/src/lecture_file.rs

```diff
@@ -0,0 +1,97 @@
+use sudo_test::{Command, Env};
+use crate::{Result, SUDOERS_ROOT_ALL, USERNAME, SUDOERS_NEW_LECTURE, SUDOERS_NEW_LECTURE_USER};
+
+#[test]
+fn default_lecture_message() -> Result<()> {
+    let expected = format!(
+        "\nWe trust you have received the usual lecture from the local System\nAdministrator. It usually boils down to these three things:\n\n    #1) Respect the privacy of others.\n    #2) Think before you type.\n    #3) With great power comes great responsibility."
```

Comment:
again: this default lecture message could be put in a variable in `src/lib.rs` to be shared across modules